rsocket-core/src/main/java/io/rsocket/core/LeaseConfig.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2015-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.rsocket.core;
+
+import io.rsocket.lease.LeaseSender;
+import io.rsocket.plugins.RequestInterceptor;
+import reactor.core.publisher.Flux;
+import reactor.util.concurrent.Queues;
+
+public final class LeaseConfig {
```

Comment:
I think `LeaseConfig` would be better in the `lease` package next to `LeaseSender` and `Lease`.